WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/SystemStatusReport/SystemStatusReportViewModel.swift

```diff
@@ -41,8 +41,187 @@ final class SystemStatusReportViewModel: ObservableObject {
 }
 
 private extension SystemStatusReportViewModel {
+    /// Format system status to match with Core's report.
+    /// Not localizing content and keep English by default.
+    ///
     func formatReport(with systemStatus: SystemStatus) -> String {
-        // TODO: handle formatting
-        return "### WordPress Environment ###"
+        var lines = ["### System Status Report generated via the WooCommerce iOS app ###"]
+
+        // Environment
+        if let environment = systemStatus.environment {
+            lines.append(contentsOf: [
+                "\n### WordPress Environment ###\n",
+                "WordPress addresss (URL): \(environment.homeURL)",
+                "Site address (URL): \(environment.siteURL)",
+                "WC Version: \(environment.version)",
+                "Log Directory Writable: \(environment.logDirectoryWritable.stringRepresentable)",
+                "WP Version: \(environment.wpVersion)",
+                "WP Multisite: \(environment.wpMultisite)",
```

Comment:
nit: maybe can use `stringRepresentable` to match core?

```suggestion
                "WP Multisite: \(environment.wpMultisite.stringRepresentable)",
```

Reply:
Great catch 😄 Fixed in 37b09e083a47c4231895df7000ecc0c106cf8e17.